### PR TITLE
Updating Portworx chart to the latest version

### PR DESCRIPTION
**Pull request by hrishipx**
Updating the Rancher Portworx chart to the latest version of Portworx.

### CODE REVIEW

File: charts/portworx/v1.0.0/templates/portworx-lighthouse.yaml [hunk 1]
```diff
@@ -11,9 +11,18 @@ metadata:
...
+    resources: ["secrets"]
+    verbs: ["get", "create", "update"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get", "create", "update"]
+  - apiGroups: [""]
+    resources: ["nodes", "services"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["stork.libopenstorage.org"]
+    resources: ["clusterpairs"]
```
Comment (pault84) [inline charts/portworx/v1.0.0/templates/portworx-lighthouse.yaml]: this needs "migrations" and "groupvolumesnapshots" aswell.

Comment (pault84) [reply charts/portworx/v1.0.0/templates/portworx-lighthouse.yaml]: Also this needs to become ClusterRole and ClusterRoleBinding see latest spec form install.portworx.com/2.0

File: charts/portworx/v1.0.0/templates/portworx-service.yaml [hunk 2]
```diff
@@ -8,8 +8,20 @@ metadata:
...
     name: portworx
+  type: NodePort
   ports:
     - name: px-api
       protocol: TCP
       port: 9001
-      targetPort: 9001
\ No newline at end of file
+      targetPort: 9001
+      nodePort: 30901
```
Comment (pault84) [inline charts/portworx/v1.0.0/templates/portworx-service.yaml]: nodePorts should be removed. we can cause conflicts by hardcoding them.

File: charts/portworx/v1.0.0/values.yaml [hunk 3]
```diff
@@ -25,13 +25,15 @@ customRegistryURL:
...
+secretType: k8s                      # Defaults to None, but can be AWS / KVDB / Vault.
 envVars: none                         # NOTE: This is a ";" seperated list of environment variables. For eg: MYENV1=myvalue1;MYENV2=myvalue2
 stork: true                           # Use Stork https://docs.portworx.com/scheduler/kubernetes/stork.html for hyperconvergence.
-storkVersion: 1.2.0
+storkVersion: 2.0.1
 
-lighthouse: false
-lighthouseVersion: 1.4.0
+lighthouse: true
+lighthouseVersion: 2.0.1
```
Comment (pault84) [inline charts/portworx/v1.0.0/values.yaml]: 2.0.2 is the latest

File: charts/portworx/v1.0.0/values.yaml [hunk 4]
```diff
@@ -43,4 +45,4 @@ serviceAccount:
 
 deploymentType: oci                     # accepts "oci" or "docker"
 imageType: none                         #
-imageVersion: 1.5.1                  # Version of the PX Image.
+imageVersion: 2.0.2                  # Version of the PX Image.
```
Comment (pault84) [inline charts/portworx/v1.0.0/values.yaml]: 2.0.2.3 is the latest.

**Comment by pault84** [comment]
This PR can be closed i just opened a new one.

**Comment by harsh-px** [comment]
@deniseschannon can you close this PR? As @pault84 commented above, this is superseded by a more recent PR.

**Pull request closed without merging**